ui/app/src/Views/Shared/AlertCompact.elm

```diff
@@ -1,16 +1,99 @@
-module Views.Shared.AlertCompact exposing (view)
+module Views.Shared.AlertCompact exposing (annotation, annotationsButton, generatorUrlButton, titleView, view)
```

Comment:
These additional functions should probably be moved to a different shared file in this directory. They're being exported because the alertlist is now importing them.

Reply:
Updated